test/unit/app/sessionStoreTest.js

```diff
@@ -159,6 +159,50 @@ describe('sessionStore unit tests', function () {
   })
 
   describe('cleanPerWindowData', function () {
+    it('handles being passed falsey input (adding frames)', function () {
+      const result = sessionStore.cleanPerWindowData()
+      assert.deepEqual(result.toJS(), {frames: []})
+    })
+    it('throws an error when input is mutable', function () {
+      assert.throws(() => {
+        sessionStore.cleanPerWindowData({windowStuff: 'demo'})
+      },
+      /undefined == true/,
+      'did not throw with expected message')
+    })
+    describe('when closed frames exist', function () {
+      it('works when closeFrames is empty', function () {
+        sessionStore.cleanPerWindowData(
```

Comment:
no result comparison?

Reply:
good call... let me clean this up a bit more...

Reply:
(I wasn't checking value since the test fails when an error happens)